README.md

```diff
@@ -1,9 +1,47 @@
+## PCL Mean and Covariance
+
 Experiments with mean and covariance computation in PCL.
 
-1. Clone this repository with submodules (pass `--recursive` flag)
+## Installation
+
+1. **Clone this repository** 
+
+   Clone with submodules and pass the `--recursive` flag:
+
+       $ git clone https://github.com/taketwo/pcl-mean-and-covariance.git --recursive-submodules
+
+2. **Configure with CMake**
+    
+       $ cd <path to>/pcl-mean-and-covariance
+       $ cmake -C CMakeLists.txt -G "Unix Makefiles"
```

Comment:
This will result in an in-source build, which is undesirable. The common practice with CMake-based projects is to create a `build` directory in the project root, cd there, and configure with `cmake ..`. This is is called out-of-source build, you could look up plenty of pages on the web explaining the advantages.